tasks/markup-1.md

```diff
@@ -1,105 +1,73 @@
 
-Использовать можно любую сеточную технологию. Можно использовать font-awesome для иконок.
+6. Fonts. You can find all the font for free (the links will be provided). The fonts should be imported properly. Don't forget about the font-weight, it also should be correct(**points - 0-10**);
 
-Препроцессоры и постпроцессоры использовать не обязательно. Отдавать на выходе файлы типа .scss, .less нельзя. Также нельзя использовать bootstrap или подобную библиотеку. Все должно быть сделано самостоятельно. штраф **минус 40 баллов**.
+7. Requirements for the blocks and columns:
+	 - The case with more text than in the markup should be considered - just copy to the column bigger text(the position of the column or layout shouldn't change, limitation by overflow)(**points - 0-10**)
+	 - You should consider the case when one more column is added. This column should appear under the first one or in the center of the next line or hidden by overflow property. It should not break the layout by being positioned as the visible third element out of the parent's borders(**points - 0-5**);
+	 -  If you are going to use inline-block property, make sure that there is no free space between the elements(**points - 0-5**);
 
-## Полезные ссылки
+8. Images in *fine ingredients* section should be square or rectangle-shaped and should be processed by CSS tools(border, border-radius)(**points - 0-10**);
 
-[Презентации](https://github.com/rolling-scopes/front-end-course/wiki/Position.-Floats.-Semantics.-CSS3)
+9. The form should be implemented with a *form* tag and with the proper tags inside of it. The input fields should be validated with basic browser validation for Email, and Name, Email and Date fields should be required. One of the field's name is incorrect, you should fix it for your application(**points - 0-10**);
 
-[Видео лекции от 06.10.2017](https://youtu.be/gYYWe7vlEQk)
+10. The footer should be placed at the bottom of the page, meaning that if all the content but the footer will be deleted from the page, the footer still will be placed at the bottom(**points - 0-5**);
 
-Ссылки на шрифты:
+11.You should implement style effects for the hover events(which effect to use is up to you - for example, the color of the navigation button can change on hover)(**points - 0-5**);
 
-[yeseva-one](https://www.fontsquirrel.com/fonts/yeseva-one)
+The header can be fixed on the gradient background when scrolling, but the additional point will not be added for implementing this feature.
 
-[gentium-basic](https://www.fontsquirrel.com/fonts/gentium-basic)
+You may use any grid markup you want. Font-awesome can be used for icons.
 
-[lato](https://www.fontsquirrel.com/fonts/lato)
+Pre-processors and post-processors are not necessary. In case of using these tools, the final file still should be in *.css* format. **Libraries such as Bootstrap are forbidden**. The layout should be implemented by the student. Fine - **40 points**.
```

Comment:
Pre-processors and post-processors are not required. In case if you decide to use these tools, the final file should still be in *.css* format. **Use of libraries such as Bootstrap is not allowed**. The whole layout should be implemented by you. Fine for non-compliance - **40 points**.